airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io.airbyte.integrations.source.mongodb/MongoDbSource.java

```diff
@@ -5,15 +5,7 @@
 package io.airbyte.integrations.source.mongodb;
 
 import static com.mongodb.client.model.Filters.gt;
-import static org.bson.BsonType.DATE_TIME;
-import static org.bson.BsonType.DECIMAL128;
-import static org.bson.BsonType.DOCUMENT;
-import static org.bson.BsonType.DOUBLE;
-import static org.bson.BsonType.INT32;
-import static org.bson.BsonType.INT64;
-import static org.bson.BsonType.OBJECT_ID;
-import static org.bson.BsonType.STRING;
-import static org.bson.BsonType.TIMESTAMP;
+import static io.airbyte.integrations.source.mongodb.MongoDbSourceUtils.*;
```

Comment:
Same as previous comment about code style that Airbyte [uses](https://airbyte.gitbook.io/airbyte/contributing-to-airbyte/code-style)